### PR TITLE
Update interpolated-string-handler.md

### PR DESCRIPTION
**Pull request by SteveDunn**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/whats-new/tutorials/interpolated-string-handler.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ This step is to build an *interpolated string handler* that recreates the curren
 - A public `AppendLiteral` method with the signature: `public void AppendLiteral(string s)`.
 - A generic public `AppendFormatted` method with the signature: `public void AppendFormatted<T>(T t)`.
 
-Internally, the builder creates the formatted string, and provides a member for a client to retrieve that string. The following code shows a `CoreInterpolatedStringHandler` type that meets these requirements:
+Internally, the builder creates the formatted string, and provides a member for a client to retrieve that string. The following code shows a `LogInterpolatedStringHandler` type that meets these requirements:
 
-:::code language="csharp" source="./snippets/interpolated-string-handler/Logger-v2.cs" id="CoreInterpolatedStringHandler":::
+:::code language="csharp" source="./snippets/interpolated-string-handler/Logger-v2.cs" id="LogInterpolatedStringHandler":::
```
Comment (BillWagner) [inline docs/csharp/whats-new/tutorials/interpolated-string-handler.md:57]: This one change should be reverted. The id is a tag in the included code file:

```suggestion
:::code language="csharp" source="./snippets/interpolated-string-handler/Logger-v2.cs" id="CoreInterpolatedStringHandler":::
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks again @SteveDunn 

There's one change that should be reverted. It's the tag for the included snippet of code from the associated project. Once you accept that change, and we get a clean build, I'll merge this.

File: docs/csharp/whats-new/tutorials/interpolated-string-handler.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Next, you'll need to update the `LogMessage` declaration so that the compiler pa
 
 :::code language="csharp" source="./snippets/interpolated-string-handler/logger-v3.cs" id="ArgumentsToHandlerConstructor":::
 
-This attribute specifies the list of arguments to `LogMessage` that map to the parameters that follow the required `literalLength` and `formattedCount` parameters. The empty string (""), specifies the receiver. The compiler substitutes the value of the `Logger` object represented by `this` for the next argument to the handler's constructor. The compiler substitutes the value of `level` for the following argument. You can provide any number of arguments for any handler you write. You add additional string arguments.
+This attribute specifies the list of arguments to `LogMessage` that map to the parameters that follow the required `literalLength` and `formattedCount` parameters. The empty string (""), specifies the receiver. The compiler substitutes the value of the `Logger` object represented by `this` for the next argument to the handler's constructor. The compiler substitutes the value of `level` for the following argument. You can provide any number of arguments for any handler you write. The arguments tat you add are string arguments.
```
Comment (SteveDunn) [inline docs/csharp/whats-new/tutorials/interpolated-string-handler.md:114]: Oops - I added a typo: "arguments tat you add"

Comment (BillWagner) [reply docs/csharp/whats-new/tutorials/interpolated-string-handler.md:114]: Thanks. Easy to fix:

```suggestion
This attribute specifies the list of arguments to `LogMessage` that map to the parameters that follow the required `literalLength` and `formattedCount` parameters. The empty string (""), specifies the receiver. The compiler substitutes the value of the `Logger` object represented by `this` for the next argument to the handler's constructor. The compiler substitutes the value of `level` for the following argument. You can provide any number of arguments for any handler you write. The arguments that you add are string arguments.
```

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks for doing this @SteveDunn 

I've approved the changes, and will merge this as soon as the build completes.

**Pull request merged**